components/sections/logoCloud/variant_a.tsx

```diff
@@ -2,21 +2,30 @@ import React from "react";
 import Image from "next/image";
 import { urlFor } from "lib/sanity";
 import { LogoCloudProps } from ".";
+import { Container } from "components/layout/Container";
+import { Heading } from "components/ui/Heading";
+import { Flex } from "components/layout/Flex/Flex";
 
 function VariantA({ title, images }: LogoCloudProps) {
   return (
     <section className="py-20">
-      <div className="container mx-auto px-4 text-center">
-        <h1 className="font-heading mb-12 text-2xl">{title}</h1>
-        <div className="-mx-2 flex flex-wrap items-center justify-center">
+      <Container>
+        <Heading className="mb-12 text-2xl text-center lg:text-2xl">
```

Comment:
Kindly conditionally render Title element to only display if it is defined/has value.